arcgis-ios-sdk-samples/Utility network/Display content of utility network container/DisplayContentUtilityNetworkContainerViewController.swift

```diff
@@ -0,0 +1,360 @@
+        // Hide the layers to display the container view.
+        (mapView.map?.operationalLayers as? [AGSLayer])?.forEach { layer in
+            layer.isVisible = false
+        }
+        // Get the corresponding features for the array of content elements.
+        utilityNetwork?.features(for: contentElements) { [weak self] (contentFeatures, error) in
+            guard let self = self else { return }
+            if let contentFeatures = contentFeatures {
+                // Create and add a symbol for each of the features.
+                contentFeatures.forEach { content in
+                    let symbol = (content.featureTable as? AGSArcGISFeatureTable)?.layerInfo?.drawingInfo?.renderer?.symbol(for: content)
+                    self.graphicsOverlay.graphics.add(AGSGraphic(geometry: content.geometry, symbol: symbol))
+                }
+                // Set the bounding box which defines the container view may be computed using the extent of the features it contains or centered around its geometry at the container's view scale.
+                if contentFeatures.count == 1,
+                   let point = contentFeatures.first?.geometry as? AGSPoint {
+                    self.mapView.setViewpointCenter(point, scale: containerElement.assetType.containerViewScale) { _ in
+                        self.presentAlert(title: nil, message: "This feature contains no associations.")
+                        guard let boundingBox = self.mapView.currentViewpoint(with: .boundingGeometry)?.targetGeometry else { return }
+                        self.identifyAssociationsWithExtent(boundingBox: boundingBox)
+                    }
+                } else {
+                    let boundingBox = AGSGeometryEngine.bufferGeometry(self.graphicsOverlay.extent, byDistance: 0.05)
+                    self.identifyAssociationsWithExtent(boundingBox: boundingBox!)
+                }
+            } else if let error = error {
+                self.presentAlert(error: error)
+            }
+        }
+    }
+    
+    /// Get associations for the specified extent and display its associations.
+    ///
+    /// - Parameter boundingBox: The gemeotry which represents the boundaries of the extent.
+    func identifyAssociationsWithExtent(boundingBox: AGSGeometry) {
+        // Add the bounding box symbol.
+        graphicsOverlay.graphics.add(AGSGraphic(geometry: boundingBox, symbol: boundingBoxSymbol.symbol))
+        let geometry = AGSGeometryEngine.bufferGeometry(graphicsOverlay.extent, byDistance: 0.05)!
+        mapView.setViewpointGeometry(geometry) { [weak self] _ in
+            guard let self = self else { return }
+            // Get the associations for this extent to display how content features are attached or connected.
+            self.utilityNetwork?.associations(withExtent: self.graphicsOverlay.extent) { containmentAssociations, _ in
+                guard let containmentAssociations = containmentAssociations else { return }
```

Comment:
If there are no associations (if there was an error), should the exit button item still become enabled?

Reply:
I think there was a previous discussion about removing the error variable because `containmentAssociations` is guarded. So even if `containmentAssociations` has 0 values, I think it would still enter the container view and the user would need an option to exit.